api/test/integration/application.e2e-spec.ts

```diff
@@ -1543,4 +1544,134 @@ describe('Application Controller Tests', () => {
       ]);
     });
   });
+
+  describe('generateLotteryResults endpoint', () => {
+    it('should generate results when no previous attempt to run lotteries has happened', async () => {
+      const unitTypeA = await unitTypeFactorySingle(
+        prisma,
+        UnitTypeEnum.oneBdrm,
+      );
+      const jurisdiction = await prisma.jurisdictions.create({
+        data: jurisdictionFactory(),
+      });
+      await reservedCommunityTypeFactoryAll(jurisdiction.id, prisma);
+      const listing1 = await listingFactory(jurisdiction.id, prisma);
```

Comment:
Can you add some multiselect questions to this listing and also update the applications in this test to have them selected/not selected? That way we can capture the multiselect logic for the lottery generation

Reply:
I went ahead and added a second test that has preferences and a mix of claimed/unclaimed to test that logic, lemme know what you think